ros_docker_images/templates/docker_images/create_ros2_image.Dockerfile.em

```diff
@@ -42,8 +42,7 @@ RUN apt-get update && apt-get install -q -y \
 RUN apt-get update && apt-get install -y \
     locales \
     && locale-gen en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LC_ALL en_US.UTF-8
+ENV LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8
```

Comment:
This is unrelated. If you'd like to propose changing this please do so in a separate PR with justification. I'll note that the best practices section of the user guide still uses the single line format: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#env